pkgs/top-level/python-packages.nix

```diff
@@ -6880,6 +6920,34 @@ in {
 
   jinja2_pluralize = callPackage ../development/python-modules/jinja2_pluralize { };
 
+  jira = buildPythonPackage rec {
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.